chain/network-primitives/src/types.rs

```diff
@@ -995,7 +995,7 @@ impl Message for QueryPeerStats {
 pub struct PartialEncodedChunkRequestMsg {
```

Comment:
should we change Borsh so that it cannot encode/decode HashSets at all ?

Reply:
@mm-near The implementation of borsh is correct, at encoding/decoding HashSets. Though it's really wasteful process.

I think encoding HashSets, should be an optional feature, turned on by default, that we can turn off.